e2e/components/Text.test.ts

```diff
@@ -44,6 +44,25 @@ test.describe('Text', () => {
       test('default @vrt', async ({page}) => {
         await visit(page, {
           id: story.id,
+          globals: {
+            featureFlags: {
+              primer_react_css_modules_team: true,
```

Comment:
Quick question: Does this mean we only run these tests with the feature flag on? Or do we run it twice, once with and once without

Reply:
> Or do we run it twice, once with and once without

Running once with it off and once with it on and comparing them to the same screenshot.

Reply:
Ooh that's smart!